lib/report_portal/cucumber/parallel_report.rb

```diff
@@ -18,12 +18,15 @@
 
 require 'parallel_tests'
 require 'sys/proctable'
+require 'fileutils'
 
 require_relative 'report'
 
 module ReportPortal
   module Cucumber
     class ParallelReport < Report
+      $folder_creation_tracking_file = Pathname(Dir.tmpdir)  + "folder_creation_tracking.lck"
```

Comment:
Is there are a reason to have global variable against instance or class variable?

Reply:
Not particularly this variable moved around a lot as I was developing I just didn't change it back. I will take care of that.